intellij-plugin/README.md

```diff
@@ -44,7 +44,7 @@ The plugin is published every Sunday in a specific **snapshots** repository. To
 configured:
 
 <kbd>Settings</kbd> > <kbd>Plugins</kbd> > <kbd>⚙</kbd>️ > <kbd>Manage Plugin
-Repositories</kbd> > <kbd>+</kbd> > <kbd>https://raw.githubusercontent.com/apollographql/apollo-kotlin/main/intellij-plugin/snapshots/plugins.xml</kbd>
+Repositories</kbd> > <kbd>+</kbd> > <kbd>https://go.apollo.dev/ij-plugin-snapshots</kbd>
```

Comment:
Reminder to call this out in the Changelog

Reply:
Good point! Also the deletion of the `snapshots/plugins.xml` file is not ideal, but it should at least show an error in the repo UI, letting users know that it's no longer valid.

Reply:
I think we can use the [`next-docs` branch](https://github.com/apollographql/apollo-kotlin/pull/5578) to start drafting the changelog. Want to put an entry there?

Reply:
perfect 👍